DQM/CTPPS/plugins/CTPPSPixelDQMSource.cc

```diff
@@ -328,19 +346,28 @@ edm::EventSetup const &)
 }
 
 //-------------------------------------------------------------------------------
-void CTPPSPixelDQMSource::beginLuminosityBlock(edm::LuminosityBlock const& lumiSeg, 
```

Comment:
Please remove both beginLuminosityBlock and endLuminosityBlock.
Add remove current_LS, you can get in analyze(), "event.getLuminosityBlock()" I think.